cmd/logcli/main.go

```diff
@@ -7,10 +7,13 @@ import (
 )
 
 var (
-	app      = kingpin.New("logcli", "A command-line for loki.")
-	addr     = app.Flag("addr", "Server address.").Default("https://log-us.grafana.net").Envar("GRAFANA_ADDR").String()
-	username = app.Flag("username", "Username for HTTP basic auth.").Default("").Envar("GRAFANA_USERNAME").String()
-	password = app.Flag("password", "Password for HTTP basic auth.").Default("").Envar("GRAFANA_PASSWORD").String()
+	app = kingpin.New("logcli", "A command-line for loki.").DefaultEnvars()
+
+	config   = app.Flag("config", "Logcli config.").Default("").String()
+	cfg      = getConfig(*config)
```

Comment:
I don't see where this is used?  What am I missing?

Reply:
cfg value use in the following code.